app/views/start.html

```diff
@@ -10,34 +10,49 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">
-        Make a claim
+        Apply to recieve a Mathematics and Physics Teacher payment
       </h1>
     </div>
   </div>
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h2 class="govuk-heading-m">Before you start</h2>
+      <p class="govuk-body">The <a href="www.google.com" target="_blank">
```

Comment:
What's the reason for adding links to google here?

Reply:
That will link out to the actual policy in the fullness of time, we have just put on
a dummy link for now.